### PR TITLE
Fixes #180 Update app.json and add Deploy button

### LINKED ISSUE #180: Add a `Deploy to Heroku` button for one-click deployment to Heroku
### This is a
- [ ] Bug report
- [x] Feature request

### Description

Currently, the only way possible to deploy yaydoc to Heroku is to manually set up yaydoc with configurations before pushing it to Heroku. In https://github.com/fossasia/yaydoc/pull/124, we added an `app.json` file. Following that, a `Deploy to Heroku` button can be added for Yaydoc

### PR DESCRIPTION
**Pull request by imujjwal96**
## Related Issue

issue #180 

## Screenshots (if appropriate):
image
image

## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -10,11 +10,12 @@
   "repository": "https://github.com/fossasia/yaydoc",
   "env": {
     "PYPANDOC_PANDOC": {
-      "description": "Set path for pandoc.",
+      "description": "PATH for pandoc. DO NOT CHANGE",
       "value": "~/vendor/pandoc/bin/pandoc"
     },
     "CALLBACKURL": {
-      "value": "/callback"
+      "value": "https://<app-name>.herokuapp.com/callback",
```
Comment (pri22296) [inline app.json]: Is it possible to make this relative?

Comment (imujjwal96) [inline app.json]: asked the same question to @sch00lb0y  and he said no.

File: app.json [hunk 2]
```diff
@@ -29,6 +30,11 @@
     },
     "ON_HEROKU": {
       "value": "true"
+    },
+    "ENCRYPTION_KEY": {
+      "value": "ABCGDHEUSHFKIRMJSDFIUTEHFNGOWPOR",
```
Comment (sch00lb0y) [inline app.json]: instead of giving a default value. is there any way to ask it from the user ?

Comment (imujjwal96) [inline app.json]: These values are editable by the user.

Comment (imujjwal96) [inline app.json]: You may test this by clicking the `Deploy to Heroku` button in https://github.com/imujjwal96/yaydoc/tree/issue180 's Readme file

**Pull request merged**